src/mozilla_sec_eia/ex_21/extractor.py

```diff
@@ -98,11 +100,12 @@ def _get_id_label_conversions():
     return id2label, label2id
 
 
-def load_test_train_set(processor: AutoProcessor, test_size: float):
+def load_test_train_set(
+    processor: AutoProcessor, test_size: float, ner_annotations: list[dict]
+):
     """Load training/test set and prepare for training or evaluation."""
     id2label, label2id = _get_id_label_conversions()
     # Cache/prepare training data
-    ner_annotations = format_as_ner_annotations()
```

Comment:
I took this out and moved the formatting of NER annotations into `train_model`. Felt like a more flexible order of operations.

Reply:
Makes sense!